core/src/test/java/io/github/wysohn/triggerreactor/core/script/interpreter/TestInterpreter.java

```diff
@@ -770,6 +770,42 @@ public void testTryCatchFinally2() throws Exception {
         verify(executor, times(3)).execute(any(), anyMap(), any(), any());
     }
 
+    @Test
+    public void testTryCatchInvokedMethod() throws IOException, ParserException, LexerException, InterpreterException {
+        Charset charset = StandardCharsets.UTF_8;
+        String text = "" +
+                "import java.io.FileReader;" +
+                "import java.lang.AssertionError;" +
+                "" +
+                "TRY;" +
+                "    FileReader(\"./no-exist-file.data\");" +
+                "    #VERIFY false;" +
+                "CATCH e;" +
+                "    IF e.getClass() == AssertionError;" +
+                "        #VERIFY false;" +
+                "    ENDIF;" +
+                "" +
+                "    #VERIFY true;" +
+                "ENDTRY";
+
+        Lexer lexer = new Lexer(text, charset);
+        Parser parser = new Parser(lexer);
+
+        Node root = parser.parse();
+
+        Map<String, Executor> executorMap = new HashMap<>();
+        executorMap.put("VERIFY", (timing, vars, context, args) -> {
+            Assert.assertEquals(true, args[0]);
+            return null;
+        });
```

Comment:
Looking all good!

Can we use mock&verify pattern instead though?

Don't forget to return 'null' with when() or the default return value will be 0, so the interpreter will stop right away (cause EXIT is integer 0)

ex)
https://github.com/TriggerReactor/TriggerReactor/blob/b2c9ea077f9984f688066b0ca6e6a830d5ed2d56/core/src/test/java/io/github/wysohn/triggerreactor/core/script/interpreter/TestInterpreter.java#L555